Makefile

```diff
@@ -25,8 +25,8 @@ Check_Targets := \
 	check-staticcheck \
 	check-misspell \
 	check-unconvert \
-	check-unparam \
 	check-ineffassign \
+#	check-unparam \
```

Comment:
you should probably fix the things the linters are complaining about, not comment them out. If you do want to remove the linter, remove the line, don't comment it out.

Reply:
This was because unparam was complaining about translate.go and the warnings `[]string` never being used. I should have put a TODO there with a note saying I would undo it as soon as I get to doing real translation.